doc/admin.md

```diff
@@ -49,4 +49,280 @@ Finally, we execute the commando package-import for the murano client library (c
 
 It will update the Murano core library with the new FIWARE extensions.
 
+## Sanity check procedures
+
+The Sanity Check Procedures are the steps that a System Administrator
+will take to verify that an installation is ready to be tested. This is
+therefore a preliminary set of tests to ensure that obvious or basic
+malfunctioning is fixed before proceeding to unit tests, integration
+tests and user validation.
+
+### End to End testing
+
+Although one End to End testing must be associated to the Integration
+Test, we can show here a quick testing to check that everything is up
+and running. For this purpose we send a request to our API in order to
+test the credentials that we have from then and obtain a valid token to
+work with.
+
+In order to make a probe of the different functionalities related to the
+Murano, we start with the obtention of a valid token for a
+registered user. Due to all operations of the Murano are using
+the security mechanism which is used in the rest of the cloud component,
+it is needed to provide a security token in order to continue with the
+rest of operations. For this operation we need to execute the following
+curl sentence.
+
+    curl -d '{"auth": {"tenantName": $TENANT,
+    "passwordCredentials":{"username": $USERNAME, "password": $PASSWORD}}}'
+    -H "Content-type: application/json" -H "Accept: application/xml"
+    http://$KEYSTONE_IP:$KEYSTONE_PORT/v2.0/tokens
+
+Both $TENANT (Project), $USERNAME and $PASSWORD must be values
+previously created in the OpenStack Keystone. The $KEYSTONE_IP and $KEYSTONE_PORT are the data of our internal
+installation of IdM (Keystone).
+
+We obtained two data from the previous sentence:
+
+-  X-Auth-Token
+
+    <token expires="2012-10-25T16:35:42Z" id="a9a861db6276414094bc1567f664084d">
+
+-  Tenant-Id
+
+    <tenant enabled="true" id="c907498615b7456a9513500fe24101e0" name=$TENANT>
+
+After it, we can check if Murano is up and running with a
+single instruction which is used to return the information of the status
+of the processes together with the queue size.
+
+    curl -v -H 'X-Auth-Token: a9a861db6276414094bc1567f664084d'
+    -X GET http://murano.lab.fiware.org:8082
+
+This operation will return the information regarding the tenant details
+of the execution of the Policy Manager
+
+    < HTTP/1.0 200 OK
+    < Date: Wed, 09 Apr 2014 08:25:17 GMT
+    < Server: WSGIServer/0.1 Python/2.6.6
+    < Content-Type: text/html; charset=utf-8
+    {
+       "versions":
+       [
+           {
+               "status": "CURRENT",
+               "id": "v1.0",
+               "links":
+               [
+                   {
+                       "href": "http://murano.lab.fiware.org:8082/v1/",
+                       "rel": "self"
+                   }
+               ]
+           }
+       ]
+    }
+
+
+For more details to use this GE, please refer to the User & Programmers Guide.
```

Comment:
Link to the User & Programmers Guide... asume the oficial OpenStack Murano web site.